src/test/scala/gen/gmos/Demo.scala

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic
+package gen.gmos
+
+import gem.Step
+import gem.enum._
+import gem.enum.GmosNorthDisperser.R831_G5302
+import gem.enum.GmosNorthFilter.GG455
+import gem.enum.GmosNorthFpu.LongSlit_0_75
+import gem.math.Wavelength
+import basic.misc._
+
+import cats.implicits._
+import cats.effect.{ ExitCode, IO, IOApp }
+import cats.effect.concurrent.Ref
+
+import fs2.Stream
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.Random
+
+/**
+ * GmosN Longslit Sequence generation demonstration.
+ */
+object Demo extends IOApp {
+
+  // A stand-in ITC that just picks from among three possible results at random.
+  val dummyItc: Itc[IO] =
+    new Itc[IO] {
+      override def calculate(
+        targetProfile: TargetProfile,
+        observingMode: ObservingMode,
+        signalToNoise: Int
+      ): IO[Itc.Result] =
+        IO {
+          Random.nextInt(3) match {
+            case 0 => Itc.Result.Success(1200 seconds, 14, 1022)
+            case 1 => Itc.Result.Success(1100 seconds, 13, 1003)
+            case _ => Itc.Result.Success(1000 seconds, 12, 1017)
```

Comment:
These are just dummy results that I invented for the purpose of showing that the exposure time could change.